src/core/WPE.js

```diff
@@ -66,10 +69,16 @@ export default class WPE {
     _noConnectionAfterTime() {
         console.log('_noConnectionAfterTime')
         if (this._state === this.STATES.NOIP) {
-            if (this._wifiPlugin === undefined)
-                this._updateUIState('NoConnection');
-            else
-                this._checkAvailableWifiConfigs()
+            if (this._wifiPlugin === undefined) {
+               if (this._wifiRetry < WIFI_MAX_RETRY_COUNT) {
+                   setTimeout(this._noConnectionAfterTime.bind(this), WIFI_PLUGIN_ACTIVATION_TIMEOUT);
```

Comment:
I think we need to move this outside the this_wifiPlugin === undefined check

Reply:
This retry is only required if the wifi is not defined, so that I think it better to keep inside this check